.gitignore

```diff
@@ -6,3 +6,6 @@ reports/
 node_modules/
 .test_durations
 yarn-error.log 
+flake8.log
+*egg-info/
```

Comment:
What is this from?

Reply:
The log is auto-generated by flake8 when it runs
Installing Flake8-vyper-linter via the commandline ended up creating a "flake8...egg-info" file, so I added it to ignore.
That line would be safe to remove, since the egg files are transient.